app/Http/Controllers/Api/V1/CurrikiGo/CourseController.php

```diff
@@ -42,6 +42,7 @@ public function __construct(LmsSettingRepositoryInterface $lmsSettingRepository)
     }
 
     /**
+     * Fetch a Course
```

Comment:
Add new line between title and description.

Reply:
Issue still persists.

Reply:
Done!